test/example.test.js

```diff
@@ -1,18 +1,18 @@
 // IMPORT MODULES under test here:
-// import { example } from '../example.js';
+import { renderToDo } from '../render-utils.js';
 
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('a complete todo should render a list with a class', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
+    const expected = `<li class="complete">Walk dog</li>`;
     
     //Act 
     // Call the function you're testing and set the result to a const
-    const actual = true;
+    const actual = renderToDo({ description: 'Walk dog', complete: true });
 
     //Expect
     // Make assertions about what is expected versus the actual result
-    expect.equal(actual, expected);
+    expect.equal(actual.outerHTML, expected);
 });
```

Comment:
Awesome test here!! Testing will come in handy in future modules. Super excited to see your continued progression!